src/rf/apps/core/models.py

```diff
@@ -222,16 +232,10 @@ class LayerImage(Model):
         default=0,
         help_text='The order which images are layered (starting from 0)'
     )
-    thumb_small = URLField(
-        null=True,
-        blank=True,
-        help_text='80x80 pixels',
-    )
-    thumb_large = URLField(
-        null=True,
-        blank=True,
-        help_text='300x300 pixels',
-    )
+    thumb_small_key = CharField(max_length=255, blank=True, default='',
```

Comment:
True but either way it will be incorrect if we end up changing sizes and the data we store is not actually connected to the size.